.ci.yaml

```diff
@@ -5892,7 +5918,7 @@ targets:
           {"dependency": "vs_build", "version": "version:vs2019"}
         ]
       shard: tool_integration_tests
-      subshard: "2_6"
```

Comment:
So this shard was running the wrong tests before...

Reply:
Whoops! I'll keep an eye on the dashboard to see if any of the tests that weren't previously running give us any problems.